src/com/lesson07/concurrentcollections/CopyOnWrite.java

```diff
@@ -0,0 +1,28 @@
+package com.lesson07.concurrentcollections;
+
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public class CopyOnWrite extends Thread {
+
+    private static CopyOnWriteArrayList<String> list = new CopyOnWriteArrayList<>();
+
+    public void run() {
+        list.add("D");
+    }
+
+    public static void main(String[] args)
+            throws InterruptedException {
+        list.add("A");
+        list.add("B");
+        list.add("C");
+
+        CopyOnWrite copyOnWrite = new CopyOnWrite();
+        copyOnWrite.run();
```

Comment:
execute thread by calling method start()

Reply:
[Creating and Starting Java Threads](http://tutorials.jenkov.com/java-concurrency/creating-and-starting-threads.html#thread-subclass)